api/helpers/sdtd/parse-commands-string.js

```diff
@@ -30,9 +32,10 @@ module.exports = {
 
 
   fn: function (inputs, exits) {
-
-    let result = inputs.commands.split(';');
-    result = result.map(x => x.trim());
+    let result = split(inputs.commands, { separator: ';', quotes: ['"'] });
+    result = result
+      .map(x => x.trim())
+      .filter(x => x !== '');
```

Comment:
Not that you need to change it, but it's usually easier to write

.filter(Boolean) which will just check for truuthy value

Reply:
Ah I forgot an empty string is falsy. I rather play safe with js :D